emar.py

```diff
@@ -67,9 +67,16 @@ def run():
           break
         i += 1
 
+  response_filename = None
+  if len(newargs) > 3:
+    response_filename = create_response_file(newargs[3:], shared.get_emscripten_temp_dir())
+    newargs = newargs[:3] + ['@' + response_filename]
+
   try:
     return shared.run_process(newargs, stdin=sys.stdin).returncode
   finally:
+    if response_filename:
```

Comment:
can add `response_filename` to `to_delete` later up instead of these two lines

Reply:
oh yea, that's simpler, updated.